.github/workflows/ci-schedule-imagescanning.yaml

```diff
@@ -0,0 +1,162 @@
+name: image-scanning-on-schedule
+on:
+  schedule:
+    # Run this workflow "At 20:00 UTC on Sunday"
+    - cron: '0 20 * * 0'
+
+jobs:
+  use-trivy-to-scan-image1:
+    name: release-1.5

```

Comment:
Can we use matrix here, Is the difference between different releases just the branch number?

Reply:
Nice suggestion! I had a try before, and it indeed works. The only problem is that doing this will directly start 32 jobs, and the images of different releases will not be combined together. Referring to  https://github.com/zhzhuang-zju/karmada/actions/runs/6662307953.

Reply:
>  I had a try before, and it indeed works. The only problem is that doing this will directly start 32 jobs, and the images of different releases will not be combined together

What are the results like now?

Reply:
Refer to https://github.com/zhzhuang-zju/karmada/actions/runs/6662001229
![image](https://github.com/karmada-io/karmada/assets/81208264/dce45173-22ff-4695-8f56-777639228218)

Referring to https://github.com/zhzhuang-zju/karmada/actions/runs/6662307953.
![image](https://github.com/karmada-io/karmada/assets/81208264/64feb622-468e-477f-a38f-74eeb75fa944)

Reply:
It looks like the former is more intuitive, but there are a lot of redundant configurations, can they be minimized?

Reply:
I can try to implement it with a script to reduce duplication of code. Then we can choose between these three options.

Reply:
@XiShanYongYe-Chang  Referring to https://github.com/zhzhuang-zju/karmada/actions/runs/6692653541/job/18182268113, it is the latest display effect. The scanning results of the same version of the images are displayed together. 
![image](https://github.com/karmada-io/karmada/assets/81208264/10f68e54-13df-4322-b343-0f0651b9176c)

The disadvantage is that the results cannot be uploaded to security tab,

Reply:
This works best in my opinion compared to the previous two options and reduces redundant configurations. Thanks~ :)